documentation/src/main/wiki/HttpServiceAdapter.md

```diff
@@ -111,6 +119,19 @@ Example configuration of a [[Service Knot|ServiceKnot]]:
         "params": {
           "path": "/service/twitter/user/{header.userId}"
         }
+      },
+      {
+        "name" : "javabooks",
```

Comment:
An example is presented [on the blog](http://knotx.io/blog/hello-rest-service/). However, it only shows how to use a fixed query parameter. 
It would be beneficial to create a new demo page explaining how to use parametrized calls. I will extend the demo with such page.